docs/pages/guides/authentication.mdx

```diff
@@ -171,6 +172,141 @@ export default function App() {
+import * as React from 'react';
+import { makeRedirectUri, useAuthRequest, useAutoDiscovery } from 'expo-auth-session';
+import { Button } from 'react-native';
+/* @info Install the Beyond Identity Embedded SDK. This can only be used in standalone and bare workflow native apps. This SDK cannot be used in the Expo Go app. */
+import { Embedded } from '@beyondidentity/bi-sdk-react-native';
+/* @end */
+
+export default function App() {
+  // Endpoint
+  const discovery = useAutoDiscovery(`https://auth-${REGION}.beyondidentity.com/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/applications/${APPLICATION_ID}`);
+  // Request
+  const [request, response, promptAsync] = useAuthRequest(
+    {
+      clientId: `${CLIENT_ID}`,
+      scopes: ['openid'],
+      redirectUri: makeRedirectUri({
+        /* @info The URI <code>[scheme]://</code> to be used in bare and standalone. If undefined, the <code>scheme</code> property of your app.json or app.config.js will be used instead. */
+        scheme: 'your.app'
+        /* @end */
+      }),
+    },
+    discovery
+  );
+
+  React.useEffect(() => {
+    const authenticate = async (url) => {
+      // Display UI for user to select a passwordless passkey
+      const passkeys = await Embedded.getPasskeys();
+
+      if (await Embedded.isAuthenticateUrl(url)) {
+        // Pass url and a selected passkey ID into the Beyond Identity Embedded SDK authenticate function
+        /* @info Parse query parameters from the 'redirectUrl' for a 'code' and then exchange that code for an access token */
+        const { redirectUrl } = await Embedded.authenticate(url, passkeys[0].id);
+        /* @end */
+      }
+    };
+
+     /* @info The response does not need to be of type 'success' if it has a url. The state value is stored in a JWT in the url 'request' paramater and the url will be validated through the Beyond Identity Embedded SDK.  */
+     if (response?.url) {
+     /* @end */
+      authenticate(url)
+    }
+  }, [response]);
+
+  return (
+    <Button
+      /* @info Disable the button until the request is loaded asynchronously. */
+      disabled={!request}
+      /* @end */
+      title="Passwordless Login"
+      onPress={() => {
+        /* @info Prompt the user to authenticate in a user interaction or web browsers will block it. */
+        promptAsync();
+        /* @end */
+      }}
+    />
+  );
+}
+```
+</Tab>
+
+<Tab>
+Set your Beyond Identity [Authenticator Config][c-beyondidentity-authenticator-config]'s Invocation Type to <code>"Manual"</code>. If "Manual" is selected, an authentication URL be returned to you as part of a JSON response. No redirects are needed and will not require authentication through a web service. The result is a completley silent OAuth 2.0 authentication using Passkeys. Since the challenge is packaged as part of the authentication URL, following the URL will result in the same behavior as if an Invocation Type of "Automatic" were selected.
```

Comment:
```suggestion
- Set your Beyond Identity [Authenticator Config's](https://developer.beyondidentity.com/docs/v1/platform-overview/authenticator-config) Invocation Type to **Manual**. 
- If **Manual** is selected, an authentication URL is returned as part of a JSON response. No redirects are needed and do not require web service authentication. The result is a completely silent OAuth 2.0 authentication using Passkeys. Since the challenge is packaged as part of the authentication URL, following the URL will result in the same behavior as if an Invocation Type of **Automatic** were selected.
```

I'm not sure I understand the context here for the following terms:
- the challenge is packaged...
- following the URL will result...

Reply:
"The challenge is packaged", is basically saying the authentication challenge is part of the authentication URL. So "following", like taking that URL and pasting it into a browser will continue the authentication as if you were doing an **Automatic** authentication. 

But, the more I think about it, it might be less confusing to just remove that part. The top of the paragraph is all you need to know about why you'd want to pick **Manual**.

Reply:
Thank you!